ebu_tt_live/bindings/_ebuttdt.py

```diff
@@ -464,7 +464,7 @@ class PercentageOriginType(TwoDimSizingMixin, ebuttdt_raw.percentageOriginType):
 
 class PixelExtentType(TwoDimSizingMixin, SizingValidationMixin, ebuttdt_raw.pixelExtentType):
 
-    _groups_regex = re.compile('(?:[+-]?(?P<first>\d*\.?\d+)(?:px))\s(?:[+-]?(?P<second>\d*\.?\d+)(?:px))')
+    _groups_regex = re.compile('(?:[+]?(?P<first>\d*\.?\d+)(?:px))\s(?:[+]?(?P<second>\d*\.?\d+)(?:px))')
```

Comment:
Need to make sure that the fix for this in Part 1 doesn't say "unsigned" but says "non-negative".